File: Programmers/Python/Level1/String_handling_basics.py
def solution(s):
    answer = True
    l = len(s)
    
    for i in range(l):
        if s[i].isalpha():              #알파벳인지 확인하는 함수 isalpha()를 사용 ( 알파벳만 있는 경우 return True )
        #if s[i].isdigit() == False:    #숫자인지 확인하는 함수 isdigit()을 사용 ( 숫자만 있는 경우 return True )
            return False
    if not(l == 4 or l == 6):
        return False
    
    return answer

File: Programmers/C/Level1/String_handling_basics.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
#include <string.h>

// 파라미터로 주어지는 문자열은 const로 주어집니다. 변경하려면 문자열을 복사해서 사용하세요.
bool solution(const char* s) {
    bool answer = true;
    int len = strlen(s);
    
    if(!(len == 4 || len == 6))
    {
        answer = false;
        return answer;
    }
    
    for(int i = 0; i < len; i++)
    {
        if(!(s[i] >= 48 && s[i] <= 57))
        {
            answer = false;
        }
    }
    
    return answer;
}